File: app/src/main/java/com/elvotra/clean/data/local/model/CommentEntity.java
package com.elvotra.clean.data.local.model;

import android.arch.persistence.room.ColumnInfo;
import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity(tableName = "comments")
public class CommentEntity {

    @PrimaryKey
    @ColumnInfo(name = "comments_id")
    private int id;

    @ColumnInfo(name = "post_id")
    private int postId;

    @ColumnInfo(name = "name")
    private String name;

    @ColumnInfo(name = "email")
    private String email;

    @ColumnInfo(name = "body")
    private String body;

    public CommentEntity(int id, int postId, String name, String email, String body) {
        this.id = id;
        this.postId = postId;
        this.name = name;
        this.email = email;
        this.body = body;
    }

    public int getId() {
        return id;
    }

    public int getPostId() {
        return postId;
    }

    public String getName() {
        return name;
    }

    public String getEmail() {
        return email;
    }

    public String getBody() {
        return body;
    }

    @Override
    public String toString() {
        return "CommentEntity{" +
                "id=" + id +
                ", postId=" + postId +
                ", name='" + name + '\'' +
                ", email='" + email + '\'' +
                ", body='" + body + '\'' +
                '}';
    }
}

File: app/src/main/java/com/elvotra/clean/data/local/model/CommentEntity.kt
package com.elvotra.clean.data.local.model

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity(tableName = "comments")
class CommentEntity(@field:PrimaryKey
                    @field:ColumnInfo(name = "comments_id")
                    val id: Int, @field:ColumnInfo(name = "post_id")
                    val postId: Int, @field:ColumnInfo(name = "name")
                    val name: String, @field:ColumnInfo(name = "email")
                    val email: String, @field:ColumnInfo(name = "body")
                    val body: String) {

    override fun toString(): String {
        return "CommentEntity{" +
                "id=" + id +
                ", postId=" + postId +
                ", name='" + name + '\''.toString() +
                ", email='" + email + '\''.toString() +
                ", body='" + body + '\''.toString() +
                '}'.toString()
    }
}
